src/chi/themes/lumen/_variables.scss

```
@@ -1292,9 +1300,15 @@ $header-navbar-tab-xs-link-font-weight: $font-weight-semi-bold;
 $header-navbar-tab-xs-link-padding: 0.5rem 1rem;
 $header-navbar-tab-xs-link-hover-text-decoration: none;
 $header-height: 3.5rem;
+$header-lg-brand-flex-grow: none;
```

Maybe `unset` would be a more suitable value for this case, since this is a new variable? Else set 0 as in all other themes (except for test):

```suggestion
$header-lg-brand-flex-grow: 0;
```